plugins/CMakeLists.txt

```diff
@@ -15,4 +15,7 @@ if(JPEGXL_ENABLE_PLUGIN_GIMP210)
   add_subdirectory(gimp)
 endif()
 
-
+option(JPEGXL_ENABLE_PLUGIN_MIME "Enable image/jxl declaration for shared-mime-info" ON)
```

Comment:
I wish the default was OFF here - there are more distros w/ recent shared-mime-info where this now has to be explicitly disabled, than the other way around...